modules/channel-smooch/src/backend/typings.ts

```diff
@@ -0,0 +1,77 @@
+import { SmoochClient } from './client'
+
+export type Clients = { [botId: string]: SmoochClient }
+
+export type Webhook = {
```

Comment:
Prefer interface when possible (not much difference, but to keep the codebase consistent).  

```
export interface Webhook {
 ...
}